src/lamp_py/bus_performance_manager/pipeline.py

```diff
@@ -49,8 +49,8 @@ def iteration() -> None:
         process_logger = ProcessLogger("event_loop")
         process_logger.log_start()
         try:
-            write_bus_metrics()
-            start_bus_parquet_updates()
+            if write_bus_metrics():
```

Comment:
Returning `bool` from here is necessary. 

`write_bus_metrics` can create parquet files for multiple days. With this change, an error on one day will preclude valid updates from another day being reflected in the Tableau datasets. 

`continue` or Excepting out of a specific `service_date` in `write_bus_metrics` is sufficient protection from polluting the export data.